### ISSUE TITLE
Taskbar: waybar exits (crashes?) if no icon is found

### ISSUE THREAD

**Issue by kanyck** [Alexays/Waybar#734]
If there is no corresponding icon in chosen theme it prints error message and terminates.

**Comment by l3nkz** [comment]
Hi, again can you share your config? What is the exact message that waybar spits out? Because actually it should only display a warning and just don't show an icon at all if it can't find any.

**Comment by kanyck** [comment]
Hi! The time has passed and now I can't reproduce this issue to provide you the info. Looks like it works... Closing for now, if I see this behavior again, I'll reopen the issue. Thanks!

**Comment by masm11** [comment]
Here is the same problem.

Whenever I startup dconf-editor, waybar disappears.

Output from waybar is:
```
Gdk-Message: 00:35:39.424: Error reading events from display: No such file or directory
```

My config is attached:
[config.gz](https://github.com/Alexays/Waybar/files/5612397/config.gz)

**Comment by masm11** [comment]
I'm sorry, I confirmed it has been fixed by 67d54ef3d53e5257e465ec556177ea569100263e.
